### PR TITLE
[Docs] Mesheryctl Docs Update fixes #10503

### LINKED ISSUE #10503: [mesheryctl] `registry` docs published under `exp` when command has been promoted already
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

### Current Behavior
Docs for `mesheryctl registry` are published at https://docs.meshery.io/reference/mesheryctl#experimental-features

### Desired Behavior

Should point to https://docs.meshery.io/reference/mesheryctl/system/registry

### Environment

- Host OS: Mac 
- Meshery Client Version: stable-v0.7.26

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- 📺 [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- ⌨️ [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- ⌨️ [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by westcalcote**
Signed-off-by: West Calcote <westcalcote@gmail.com>

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 879d8d067666059145fa67ef72fb961680d7459a at: https://65e795158eccdc73d147bf3a--meshery-docs-preview.netlify.app

File: mesheryctl/internal/cli/root/app/app.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var (
 var AppCmd = &cobra.Command{
 	Use:   "app",
 	Short: "Cloud Native Apps Management",
-	Long: `All apps operations: import, list, view, onboard and offboard.
+	Long: `"app" command is being deprecated and moved to "design" command. All apps operations: import, list, view, onboard and offboard.
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/app/app.go:41]: Thanks for including this.

**Review by leecalcote — DISMISSED** [review]
LGTM

**Comment by l5io** [comment]
🚀 Preview for commit a63b92604c4a43810d653ef342fc7ab2b3c864f3 at: https://65e79b39d1619c798b61fcba--meshery-docs-preview.netlify.app

**Pull request merged**